packages/calcite-components/src/components/card/card.scss

```diff
@@ -8,6 +8,15 @@
  * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
  * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
  *
+ * @prop --calcite-card-checkbox-background-color: defines the background-color of the sub-component.
```

Comment:
Let's add all of these checkbox ones in as deprecated. We can pick up the new Card tokens for the updated behavior (an internal action with mapped tokens) when `main` gets merged in to `7180`.